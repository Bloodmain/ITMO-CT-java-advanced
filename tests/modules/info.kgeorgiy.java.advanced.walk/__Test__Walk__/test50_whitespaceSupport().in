__Test__Walk__/test50_whitespaceSupport()/  _  _ _  _     _  __  _  ___ 
__Test__Walk__/test50_whitespaceSupport()/  __  __ _ _    _ __   ___    
__Test__Walk__/test50_whitespaceSupport()/  _     __      _   _    _   _
__Test__Walk__/test50_whitespaceSupport()/   __ ___    _  ___ _   _     
__Test__Walk__/test50_whitespaceSupport()/_    ______   _           _ __
__Test__Walk__/test50_whitespaceSupport()/ __  __      _ _ ____       __
__Test__Walk__/test50_whitespaceSupport()/  _     ____ _ __   __ __   _ 
__Test__Walk__/test50_whitespaceSupport()/_  _ _    _       __   __ __  
__Test__Walk__/test50_whitespaceSupport()/___  _ _  _  _  __   _ _   ___
__Test__Walk__/test50_whitespaceSupport()/ _  _  __  _    _    _    _   
