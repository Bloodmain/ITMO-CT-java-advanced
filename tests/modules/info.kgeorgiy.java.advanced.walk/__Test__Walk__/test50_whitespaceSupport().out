55538cc5 __Test__Walk__/test50_whitespaceSupport()/  _  _ _  _     _  __  _  ___ 
396d3551 __Test__Walk__/test50_whitespaceSupport()/  __  __ _ _    _ __   ___    
7d4b7e70 __Test__Walk__/test50_whitespaceSupport()/  _     __      _   _    _   _
8e92e51b __Test__Walk__/test50_whitespaceSupport()/   __ ___    _  ___ _   _     
db26632e __Test__Walk__/test50_whitespaceSupport()/_    ______   _           _ __
611b3841 __Test__Walk__/test50_whitespaceSupport()/ __  __      _ _ ____       __
b26d818b __Test__Walk__/test50_whitespaceSupport()/  _     ____ _ __   __ __   _ 
ded21845 __Test__Walk__/test50_whitespaceSupport()/_  _ _    _       __   __ __  
235ad359 __Test__Walk__/test50_whitespaceSupport()/___  _ _  _  _  __   _ _   ___
b57543ca __Test__Walk__/test50_whitespaceSupport()/ _  _  __  _    _    _    _   
